intro-Globus/ch_FileManager.tex

```diff
@@ -0,0 +1,42 @@
+\chapter{Managing and transferring files}
+
+\section{The File Manager}\label{file-manager}
+
+After you have signed up and logged in to Globus, you will begin at the \emph{File Manager}. The first time you use the \emph{File Manager}, all fields will be blank.
+
+\begin{center}
+\includegraphics[width=0.5\textwidth]{img/filemanager-1.png}
+\end{center}
+
+\section{Collections and Endpoints}\label{collections-endpoints}
+
+A \gls{Collection} is a named location containing data you can access with Globus. Collections can be hosted on many different kinds of systems, including campus storage, HPC clusters, laptops, Amazon S3 buckets, Google Drive, and scientific instruments. When you use Globus, you do not need to know a physical location or details about storage. You only need a collection name. A collection allows authorized Globus users to browse and transfer files. Collections can also be used for sharing data with others, for data publication, and for enabling discovery by other Globus users. \gls{Globus Connect} is used to host collections.
```

Comment:
this is copy/paste from the gloassary. i would not do that, it is very redundant.